src/lib/components/BlogPage.svelte

```diff
@@ -0,0 +1,159 @@
+<script lang="ts">
+  import { createEventDispatcher } from "svelte";
+  import Button from "./Tag.svelte";
+  import BlogLink from "$lib/components/BlogLink.svelte";
+  import type { Tags } from "$lib/utils/types/microcms";
+
+  export let more: boolean = false;
+  export let blogs;
+  export let limit: number = 10;
+
+  let blogListContents = blogs.blogList.contents;
```

Comment:
`import type { EndPoints } from "$lib/utils/types/microcms";` を追加した上で,

```suggestion
  export let blogs: EndPoints["gets"]["blogs"];
  export let limit: number = 10;

  let blogListContents = blogs.contents;
```

としてあげると, microCMS が返す型が流れてきます！  そのおかげで L30 や L33 などの型注脚が不要になります. 🎉

Reply:
こちらに修正したところ、
```patch
32    filterTag === "All" || blog.tags.includes(filterTag),
```
の部分で"All"が入るから形ちがうぞって言われたので、
```patch
- 32    filterTag === "All" || blog.tags.includes(filterTag),
+ 32    filterTag === "All" || blog.tags.includes(filterTag as Tags),
```
としてfilterTagにも型を（All入るけど）指定しました